packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1048,9 +1048,9 @@ oie.consent.scopes.enduser.title = would like to:
 oie.consent.enduser.accept.label = Yes, it's me
 oie.consent.enduser.deny.label = No, it's not me
 oie.consent.enduser.title = Did you just try to sign in?
-oie.consent.enduser.email.allow.description = To continue, return to the device or window where you requested the email link. You may close this window at any time.
-oie.consent.enduser.deny.description = You may close this window at any time.
-oie.return.to.original.tab = Please return to the original tab.
+oie.consent.enduser.email.verified.title = Email verified!
+oie.return.to.original.tab = Please <span style="font-weight:bold">return</span> to your original screen to continue.
```

Comment:
Did UX propose this? Wrapping `return` with text emphasis without a click handler seems like it'll confuse users.

I'd suggest making `return` a link or removing the `span` element.

Reply:
UX Proposal: https://oktainc.atlassian.net/browse/OKTA-402915?focusedCommentId=2046134

`return` is just being bolded not a link since it is telling user to return to the original tab not navigating them from the current window.